VirtualEconomyFramework/VEDriversLite/NFT/NFTHelpers.cs

```diff
@@ -804,9 +804,12 @@ public static async Task<string> MintNFT(string address, EncryptionKey ekey, INF
             };
 
             try
-            {
+            {                               
                 // send tx
-                var rtxid = await NeblioTransactionHelpers.MintNFTTokenAsync(dto, ekey, nutxos, tutxos);
+                var transaction = await NeblioTransactionHelpers.MintNFTTokenAsync(dto, ekey, nutxos, tutxos);
+
+                var rtxid = await NeblioTransactionHelpers.SignAndBroadcastTransaction(transaction, secret);

```

Comment:
Please move these calls to the NeblioAccountBase instead of this NFTHelpers class.

Reply:
Tomas which calls do you mean exactly. The MintNFTTokenAsync and SignAndBroadcastTransaction (or) MintNFT.

Reply:
sorry for not clear description. I agree it is confusing. I meant the SignAndBroadcastTransaction. The signing should be in layer of the Account.